PSWsl.psm1

```diff
@@ -157,12 +157,12 @@ function Get-WslDistribution {
     
     begin {
         # parse
-        $array = [System.Text.Encoding]::Unicode.GetString([System.Text.Encoding]::UTF8.GetBytes((wslconfig.exe /l))).Split("   ", [System.StringSplitOptions]::RemoveEmptyEntries) | Select-Object -Skip 1 -Unique
-
+        $array = [System.Text.Encoding]::Unicode.GetString([System.Text.Encoding]::UTF8.GetBytes((wslconfig.exe /l))) -split '\s\s+' | Select-Object -Skip 1 -Unique
+        $array = $array[0..($array.length-2)]
```

Comment:
Just curious... Why -2? I would expect -1.

Could you add a comment here?

Reply:
This is to trim the extra item in the array, I do not know why but there is a extra item inside `$array` after line 160, so line 161 is basically removing that item at the end from the `$array`.

Reply:
I see. The extra item must be an empty space.

When you say you tried Pipeline, do you mean:

```
Get-WSLDistribution
```

Piping into something? Or piping something into it?

Reply:
Well... I mean using pipeline on line 160 so that line 161 can be removed, the pipeline for `Get-WSLDistribution` works anyway

Reply:
No worries :) how it is now is good!